include/boost/json/string.hpp

```diff
@@ -1061,8 +1148,12 @@ class string
 
     //------------------------------------------------------
 
-    /** Return a pointer to the storage associated with the container
+    /** Access the storage associated with the container.
 
+        Returns a pointer to the storage associated with the container
```

Comment:
If you want to avoid the repetition from the brief you can start this sentence with "This function returns..." or you can switch to the passive voice "The storage associated with the container is returned."